src/main/scala/ai/privado/utility/ImportUtility.scala

```diff
@@ -0,0 +1,106 @@
+package ai.privado.utility
+
+import scala.util.control.Breaks._
+import java.nio.charset.MalformedInputException
+import scala.io.Source
+import ai.privado.model.Language
+
+import scala.collection.parallel.CollectionConverters.ImmutableIterableIsParallelizable
+
+object ImportUtility {
+
+  private var allImports: Set[String] = Set();
+
+  // get language specific import matching regex
+  private def getMatchImportRegex(language: Language.Value): String = {
+    val result = language match {
+      case Language.JAVA       => "^\\s*(import)\\s+.*$"
+      case Language.PYTHON     => "^\\s*(import)\\s+.*$"
+      case Language.JAVASCRIPT => "^\\s*(import|require)\\s+.*$"
+      case default             => "(import)\\s+" // Default is java
+    }
+    return result;
+  }
+
+  private def getFileExtension(language: Language.Value): String = {
+    val result = language match {
+      case Language.JAVA       => ".java"
+      case Language.PYTHON     => ".py"
+      case Language.JAVASCRIPT => ".js"
+      case default             => ".java" // Default is java
+    }
+
+    return result;
+  }
+
+  // get regex to match only import words
+  private def getLiteralWordRegex(language: Language.Value): String = {
+    val result = language match {
+      case Language.JAVA       => "(import|static)\\s+"
+      case Language.PYTHON     => "(import)\\s+"
+      case Language.JAVASCRIPT => "(import|require)\\s+"
+      case default             => "(import)\\s+" // Default is java
+    }
+    return result;
+  }
+
+  private def returnImportStatements(filePath: String, language: Language.Value): Set[String] = {
+    val source               = Source.fromFile(filePath)
+    val matchImportRegex     = getMatchImportRegex(language);
+    val onlyLiteralWordRegex = getLiteralWordRegex(language);
+
+    var multilineFlag              = false;
+    var uniqueImports: Set[String] = Set()
+    try {
+
+      breakable {
+        for (line <- source.getLines()) {
+          val scanLine = line.trim();
+          if (scanLine.matches("/\\*.*")) { // set flag if multiline comment is encountered
```

Comment:
The multiline logic written will work only for languages where `\*` is used for multiline statement.

Pick this up by language